packages/tables/src/styled/StyledCell.ts

```diff
@@ -19,14 +19,13 @@ export interface IStyledCellProps {
    * checkboxes or icons)
    */
   isMinimum?: boolean;
-  /** Applies truncated text styling */
+  /** Truncates long text with an ellipsis */
   isTruncated?: boolean;
-  /**
-   * Styling for a cell that contains an `OverflowButton`
-   */
+  /** Applies styling for a cell that contains an overflow menu */
   hasOverflow?: boolean;
-  /** The width of the cell */
+  /** Adjusts the width of the cell */
   width?: string | number;
+  /** Adjusts the vertical pading of the cell */
```

Comment:
*padding

Reply:
Hmm I'm not sure why they are missing. I have run yarn start:styleguidist but it actually doesn't look like it is picking up any of my props in the `styled` folder..

Reply:
Yep, I just double-checked and the rendered docs are looking good, and only https://github.com/zendeskgarden/react-components/blob/67b39a7787e3f51560a8f60d84a0d6eb21192e4b/packages/tables/src/elements/SortableCell.tsx#L20-L22 is missing.

Reply:
Forgot to push 😅